modules/nw-installing-external-dns-operator-cli.adoc

```diff
@@ -0,0 +1,160 @@
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* You are logged into the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `Namespace` object:
+
+.. Create a YAML file that defines the `Namespace` object:
++
+.Example `namespace.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: external-dns-operator
+----
+
+.. Create the `Namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f namespace.yaml
+----
+
+. Create an `OperatorGroup` object:
+
+.. Create a YAML file that defines the `OperatorGroup` object:
++
+.Example `operatorgroup.yaml` file
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: external-dns-operator
+  namespace: external-dns-operator
+spec:
+  upgradeStrategy: Default
+  targetNamespaces:
+  - external-dns-operator
+----
+
+.. Create the `OperatorGroup` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f operatorgroup.yaml
+----
+
+. Create a `Subscription` object:
+
+.. Create a YAML file that defines the `Subscription` object:
++
+.Example `subscription.yaml` file
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: external-dns-operator
+  namespace: external-dns-operator
+spec:
+  channel: stable-v1
+  installPlanApproval: Automatic
+  name: external-dns-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+.. Create the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f subscription.yaml
+----
+
+.Verification
+
+. Get the name of the install plan from the subscription:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator \
+    get subscription external-dns-operator \
+    --template='{{.status.installplan.name}}{{"\n"}}'
+----
+
+. Check the status of the install plan:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator \
+    get ip <install_plan_name> \
+    --template='{{.status.phase}}{{"\n"}}'
+----
++
+.Required output
+[source,terminal]
+----
+Complete
+----
+
+. Check the status of the `external-dns-operator` pod is running:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator get pod
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME                                     READY   STATUS    RESTARTS   AGE
+external-dns-operator-5584585fd7-5lwqm   2/2     Running   0          11m
+----
+
+. Confirm the subscription is `Red Hat Operators` catalog source:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator get subscription
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME                    PACKAGE                 SOURCE             CHANNEL
+external-dns-operator   external-dns-operator   redhat-operators   stable-v1
+----
+
+. Check the `external-dns-operator` version:
```

Comment:
```suggestion
. Check the External DNS Operator version by running the following command:
```
- Is this step verifying that the External DNS Operator version is updated to a correct value? If yes, then same comment about using "verify".
- IMO, instead of using a resource name `external-dns-operator`, how about using the Operator name?
- I don't think this command always expects the EDNS Operator version to be `1.2.0` after installing the Operator. (SME can confirm though). So considering that the version changes for every Operator release, I would leave up to you to use a variable for specifying the version.

Reply:
ty